### PR TITLE
Feature/pic

### PR DESCRIPTION
**Pull request by scolalongo**
Add gitlab pipeline to build docker image + Makefile.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,5 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+.env
```
Comment (MarcAntoine-Arnaud) [inline .gitignore]: missing line return

File: README.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+## Requirements
 
 Depends on [py_amqp_connection](https://github.com/FTV-Subtil/py_amqp_connection) package:
 ```bash
 pip3 install amqp_connection
 ```
 
-Usage
+## Usage
 -----
```
Comment (MarcAntoine-Arnaud) [inline README.md]: so remove -----

File: README.md [hunk 3]
```diff
@@ -73,3 +74,30 @@ Example of handled AMQP message body:
...
+    \----------/
+         |
+     +-------+
+     | build |
+     +-------+
+            
+
+### Docker
+
+The command `make docker-build` will build an image named `mediacloudai/rdf_worker`.
```
Comment (MarcAntoine-Arnaud) [inline README.md]: wrong image name `mediacloudai/rdf_worker`

File: .gitignore (hunk 1 above)
Comment (scolalongo) [reply .gitignore]: Done.

File: README.md (hunk 2 above)
Comment (scolalongo) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (scolalongo) [reply README.md]: Done.

**Pull request merged**